scripts/setup_python_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#
+#  Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+set -e
+
+if test "${SPARK_HOME}" == ""; then
```

Comment:
Can you rename this file to be prepare_python_tests.sh to be consistent with travis/*?